Mathlib/CategoryTheory/Skeletal.lean

```diff
@@ -134,10 +136,40 @@ lemma skeleton_isSkeleton : IsSkeletonOf C (Skeleton C) (fromSkeleton C) where
   skel := skeleton_skeletal C
   eqv := fromSkeleton.isEquivalence C
 
-section
-
 variable {C D}
 
+section API
```

Comment:
I don't think `API` is a helpful section descriptor, perhaps let's just leave it as `section`?